src/scss/jooa11y.scss

```diff
@@ -210,26 +208,27 @@ html[data-jooa11y-theme="dark"] {
   height: 100%;
   margin: -10px;
   content: "";
-  border: 4px solid rgba(255, 255, 255, .9);
+  border: 4px solid rgba(255, 255, 255, 0.9);
   border-top: 4px solid transparent;
   border-bottom: 4px solid transparent;
   border-radius: 50%;
-  animation: jooa11y-rotate 1.5s ease;
-  animation-fill-mode: forwards;
+  -webkit-animation: jooa11y-rotate 1.5s ease;
```

Comment:
no need for the webkit prefix here

Reply:
Brian,  I'm not a commit specialist and I try my best.
 I  changed the compiled CSS. That is is the isssue . The scss-File is a file without vars. So that must be the issue.
 I think you know the code best and it is easy for your to fix  wrong format.